Model/Sales/TrackTraceHolder.php

```diff
@@ -205,14 +224,13 @@ private function convertDataForCdCountry($magentoTrack)
 
             foreach ($products as $product) {
                 $myParcelProduct = (new MyParcelCustomsItem())
-                    ->setDescription($product->getName())
-                    ->setAmount($product->getQty())
-                    ->setWeight($product->getWeight() ?: 1)
-                    ->setItemValue($product->getPrice())
+                    ->setDescription($product->consignment->getName())
+                    ->setAmount($product->consignment->getQty())
+                    ->setWeight($product->consignment->getWeight() ?: 1)
+                    ->setItemValue($product->consignment->getPrice())
```

Comment:
heb je een consignment in $product?

Reply:
Het is sowieso een rare method, hij wordt nergens gebruikt

Reply:
Done